lib/config/httpRoutes.js

```diff
@@ -128,6 +128,9 @@ module.exports = [
   {verb: 'post', url: '/users/:_id/_createRestricted', controller: 'security', action: 'createRestrictedUser'},
   {verb: 'post', url: '/users/_create', controller: 'security', action: 'createUser'},
   {verb: 'post', url: '/users/:_id/_create', controller: 'security', action: 'createUser'},
+  {verb: 'post', url: '/profiles/_mdelete', controller: 'security', action: 'mDeleteProfile'},
```

Comment:
The http RFC is not clear about allowing a body in a DELETE request. To be safe, _mdelete calls are set as POST
http://stackoverflow.com/questions/299628/is-an-entity-body-allowed-for-an-http-delete-request